components/sup/src/manager/mod.rs

```diff
@@ -160,7 +160,7 @@ impl Default for ManagerConfig {
             eventsrv_group: None,
             update_url: "".to_string(),
             update_channel: "".to_string(),
-            gossip_listen: GossipListenAddr::default(),
+            gossip_listen: GossipListenAddr::configured_value(),
```

Comment:
Since the only place in real code that `ManagerConfig::default()` seems to get used is up in the CLI layer, where we're going to inject any configured `GossipListenAddr` value anyway, I think using `default()` here, rather than `configured_value()` is safe and more intention-revealing.